utils/form-schema.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+
+export const signUpSchema = z.object({
+  full_name: z.string().min(1,{message: "Name is required."}),
+  email: z.string().email({message: "Email address is invalid."}),
+  company_name: z.string().min(1),
```

Comment:
max length also here
```suggestion
  company_name: z.string()
  .min(1, { message: "Company name is required." })
  .max(100, { message: "Company name is too long." }),
```